client/state/sites/domains/test/fixture.js

```diff
@@ -34,13 +36,15 @@ export const DOMAIN_PRIMARY = {
 	isPendingIcannVerification: false,
 	manualTransferRequired: false,
 	newRegistration: false,
+	name: 'retronevergiveup.me',
+	owner: 'John Doe',
 	partnerDomain: false,
 	pendingRegistration: false,
 	pendingRegistrationTime: '',
 	isPrimary: true,
 	isPrivate: false,
 	registrationDate: 'March 9, 2016',
-	type: 'mapping',
+	type: 'MAPPED',
```

Comment:
Well, they were "correct" because it didn't really use the `getDomainType` in assembler. So correct according to assembler. But incorrect otherwise. When I tried to render the domain notice on My Plans page in #7860 with this data but without getting `type` from `getDomainType`, it wouldn't work as `getDomainType` uses some other props from the raw `domain` (ie received from API) to infer the type.

This should be correct now.